UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FulfilmentDataService.cs

```diff
@@ -26,6 +26,17 @@ public class FulfilmentDataService : IFulfilmentDataService
         private const string UPDATEZIPEXCHANGESETFILEEXTENSION = "zip";
         private const string CATALOGUEFILEEXTENSION = "xml";
         private const string ENCUPDATELISTFILEEXTENSION = "csv";
+
```

Comment:
Have just noticed a few places in this class where the logging doesn't log the Correlation ID because DateTime.Now.ToUniversalTime() is passed to the string twice, leaving the CommonHelper.CorrelationID on the end unreferenced. Please see the following lines:

- 194
- 198
- 214
- 219

As an example, line 194 is shown here:

                    _logger.LogInformation(EventIds.ExtractZipFileStarted.ToEventId(), "Extracting zip file {fileName} started at {DateTime} | _X-Correlation-ID:{CorrelationId}", file.FileName, DateTime.Now.ToUniversalTime(), DateTime.Now.ToUniversalTime(), CommonHelper.CorrelationID);

Note the {DateTime} is only in the string once, but DateTime.Now.ToUniversalTime() is in the parameter list twice.

Reply:
Removed duplicated datetime